src/overlays/actors/ovl_Bg_Hidan_Sekizou/z_bg_hidan_sekizou.c

```diff
@@ -110,25 +112,323 @@ static ColliderJntSphInit D_8088E258 = {
     6,
     D_8088E180,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Hidan_Sekizou/func_8088CEC0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Hidan_Sekizou/BgHidanSekizou_Init.s")
+CollisionCheckInfoInit D_8088E268 = { 1, 40, 240, MASS_IMMOVABLE };
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 400, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1500, ICHAIN_STOP),
+};
+
+void* sFireballsTexs[] = { gFireTempleFireball0Tex, gFireTempleFireball1Tex, gFireTempleFireball2Tex,
+                           gFireTempleFireball3Tex, gFireTempleFireball4Tex, gFireTempleFireball5Tex,
+                           gFireTempleFireball6Tex, gFireTempleFireball7Tex };
```

Comment:
trailing comma for format. also can make it static along with the colcheck info init above